src/ae_kqueue.c

```diff
@@ -37,15 +37,21 @@ typedef struct aeApiState {
     int kqfd;
     struct kevent *events;
 
+    /* changes is used to buffer incoming events that will be
+     * registered in bulk via kevent(2). */
+    struct kevent changes[1024];
+    unsigned int nch;
```

Comment:
```suggestion
    unsigned int num_changes;
```
Minor preference to make this wording clearer.